WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -923,22 +1130,22 @@ e_britannia = {
 					province = 1601
 					cultural_names = {
 						#Roman Cultures
-						name_list_roman = cn_durobrivae
-						name_list_romano_gallic = cn_durobrivae
-						name_list_romano_hispanic = cn_durobrivae
-						name_list_romano_british = cn_durobrivae
-						name_list_romano_african = cn_durobrivae
-						name_list_romano_raetian = cn_durobrivae
-						name_list_romano_thracian = cn_durobrivae
-						name_list_romano_illyrian = cn_durobrivae
-						name_list_romano_pannonian = cn_durobrivae
-						name_list_dalmatian = cn_durobrivae
-						name_list_romano_noric = cn_durobrivae
-						name_list_romano_gothic = cn_durobrivae
-						name_list_romano_dacian = cn_durobrivae
-						name_list_romano_frank = cn_durobrivae
-						name_list_romano_aramean = cn_durobrivae
-						name_list_romano_vandalic = cn_durobrivae
+						name_list_roman = cn_petroburgum
+						name_list_romano_gallic = cn_petroburgum
+						name_list_romano_hispanic = cn_petroburgum
+						name_list_romano_british = cn_petroburgum
+						name_list_romano_african = cn_petroburgum
+						name_list_romano_raetian = cn_petroburgum
+						name_list_romano_thracian = cn_petroburgum
+						name_list_romano_illyrian = cn_petroburgum
+						name_list_romano_pannonian = cn_petroburgum
+						name_list_dalmatian = cn_petroburgum
+						name_list_romano_noric = cn_petroburgum
+						name_list_romano_gothic = cn_petroburgum
+						name_list_romano_dacian = cn_petroburgum
+						name_list_romano_frank = cn_petroburgum
+						name_list_romano_aramean = cn_petroburgum
+						name_list_romano_vandalic = cn_petroburgum
```

Comment:
Durobrivae was much more signficant, Burgum is a medieval construction from Petersburgh, name which wasn't used during the early Anglo-Saxon rule

Reply:
Ok, this was not a very good move from my side. Reverted back to Durobrivae.

Reply:
I'm not sure if this is a change in your local branch or not because I don't see any revert here.

Reply:
> I'm not sure if this is a change in your local branch or not because I don't see any revert here.

So, you can't see my revert? That's quite strange, because everything is working fine for me. Here is how that section looks like on Github:
![Untitled](https://user-images.githubusercontent.com/44349204/234107089-8e31f305-831d-4baa-9339-bb71ed99738a.png)

Reply:
My bad, I see it as well now, resolving this then